File: src/frontend/src/app/components/cadastro/unidade-negocio/model/unidade-negocio-grid-dto.ts
export class UnidadeNegocioGridDTO {
  id: string;
  codigo: string;
  nome: string;
  cnpj?: string;
  ativa: boolean;
  deleted?: boolean;

  constructor(
    id: string,
    codigo: string,
    nome: string,
    cnpj: string | undefined,
    ativa: boolean,
    deleted?: boolean
  ) {
    this.id = id;
    this.codigo = codigo;
    this.nome = nome;
    this.cnpj = cnpj;
    this.ativa = ativa;
    this.deleted = deleted;
  }
}

File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/cadastro/unidadenegocio/UnidadeNegocioGridDTO.java
package br.com.grupopipa.gestaointegrada.cadastro.unidadenegocio;

import br.com.grupopipa.gestaointegrada.core.dto.GridDTO;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.util.UUID;

@Builder
@Getter
@Setter
public class UnidadeNegocioGridDTO implements GridDTO {
    private UUID id;
    private String codigo;
    private String nome;
    private String cnpj;
    private Boolean ativa;
    private Boolean deleted;
}
